### DISCUSSION TITLE
Tero Subtitler 1.0.0.0

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#51] [Announcements]
### Introducing: Tero Subtitler

Since the last update in March (under Subtitle Workshop), URUWorks’ subtitling solution has quietly undergone several changes behind the scenes, including over a hundred fixes and the implementation of new features, so we now present the first public alpha of Tero Subtitler for exploration.

_Use it at your own risk!_

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.0.0'>Tero Subtitler 1.0.0.0</a>.</em>

**Comment by chenlung** [comment]
While Tero Subtitler is an improvement on the last iteration (Subtitle Workshop), it is not yet recommended for production work, hence we welcome assistance from testers (including professionals of the subtitling industry) and translators to coders and designers, so people are encouraged to share this announcement widely. Those offering wisdom and/or pull requests should visit Issues and Discussions (ideally focusing on the higher priority ones) because the tentative plan is to issue version 1.0 of Tero Subtitler in July, the focus of which is stability and usability to help ensure solid foundations and a positive user experience upon its debut. 

There are a few weeks to check for missing features (and have them implemented, depending on their essentialness and/or simplicity) before the ensuing beta releases (of which there will be several in July), followed by release candidates (a handful in late July); all of which will help refine the application, and we encourage the testing of implemented fixes (ensuring they work as intended and haven’t resulted in regressions).

Areas that may/do require attention (though not limited to these):
- Audio to text (functionality of Whisper; post-processing)
- Drop frame/non-drop frame
- OS-specific operations (e.g. ensuring all components — including third-party — function correctly across different platforms)
- Shortcuts (are the keystrokes correct/standard? Too many defaults?)
- Timebase (e.g. handling of time codes and video preview; SMPTE, etc.)
- Text (from those with a good grasp of English)
- Translation
- UI (e.g. icon quality; adherence to guidelines/conventions)
- UX (e.g. illogical processes)
- Validation of file formats during opening and saving (i.e. are they read and written correctly, in accordance with specifications?)

**Comment by Tantacrul** [comment]
Congrats!
